src/corrector.rb

```diff
@@ -0,0 +1,9 @@
+class Corrector
+  def initialize(name)
+    @name = name
+  end
+
+  def correct_name(name)
+    name.capitalize[1..10].strip
+  end
+end
```

Comment:
- [ ] You probably don't need the `initialize()` method here, the `correct_name()` method should be enough. Kindly remove it.